### ISSUE TITLE
What optimizer is used?

### ISSUE THREAD

**Issue by tureture** [MrBlankness/LightM-UNet#19]
Hi!
I am looking to recreate some of the results of the paper and is wondering about what optimizer and learning rate was used when training the LightM-UNet on the LiTs dataset?

In the paper it says: 

> SGD was employed as the optimizer, initialized with a learning rate of 1e-4.
The PolyLRScheduler was used as the scheduler, and a total of 100 epochs were
trained

Which appears to be somewhat close to the default used in nnUnet [here.](https://github.com/MrBlankness/LightM-UNet/blob/master/lightm-unet/nnunetv2/training/nnUNetTrainer/nnUNetTrainer.py#L471)

However when looking through the [nnUNetTrainerLightMUNet.py](https://github.com/MrBlankness/LightM-UNet/blob/master/lightm-unet/nnunetv2/training/nnUNetTrainer/nnUNetTrainerLightMUNet.py#L119C36-L124) file it instead appears that Adam was used. 

Could you please clarify what optimizer and hyperparameters was used? 
Also if anyone has a good intuition why you would use one over the other when choosing between Adam or SGD, I would love to hear it!
